README.md

```diff
@@ -249,90 +249,129 @@ For the out-of-the-box implementation of capturing exceptions thrown out of your
 
 ### Installation
 
 #### With SBT
 
-Add the following line to your build.sbt's libraryDependencies:
+Add the following line to your `build.sbt`'s `libraryDependencies`:
 
 ```
 libraryDependencies ++= Seq(
-    "com.mindscapehq" % "raygun4java-play2" % "3.0.0"
+    "com.mindscapehq" % "raygun4java-play2" % "4.0.0"
 )
 ```
 
 #### With Maven
 
-Add the raygun4java-play2-3.x.x dependency to your pom.xml (following the instructions under 'With Maven and a command shell' at the top of this file).
+Add the `raygun4java-play2-4.x.x` dependency to your `pom.xml` (following the instructions under 'With Maven and a command shell' at the top of this file).
 
 ### Usage
 
-For automatic exception sending, in your Play 2 app's global error handler, RaygunPlayClient has a method which allows you to pass in a RequestHeader and send a Throwable:
+To handle exceptions automatically, define a custom error handler by creating a class that inherits from `HttpErrorHandler`. For Play 2's newer versions, `GlobalSettings.onError` is no longer available. Instead, you should use `HttpErrorHandler`.
 
-**In Scala**
-
-**app/Global.scala**
-```scala
-override def onError(request: RequestHeader, ex: Throwable) = {
-  val rg = new RaygunPlayClient("your_api_key", request)
-  val result = rg.Send(ex)
-
-  super.onError(request, ex)
-}
+1. Add the following line to your `conf/application.conf`:
+```
+play.http.errorHandler = "com.raygun.CustomErrorHandler"
 ```
 
-**In Java**
+2. Create the custom error handler class:
+
+**In Java:**
 
-**app/Global.java**
 ```java
+package com.raygun;
+
 import play.*;
 import play.mvc.*;
 import play.mvc.Http.*;
-import play.libs.F.*;
+import play.libs.concurrent.HttpExecutionContext;
+import play.http.HttpErrorHandler;
 
-import com.mindscapehq.raygun4java.play2.RaygunPlayClient;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
 
-import static play.mvc.Results.*;
+import com.mindscapehq.raygun4java.play2.RaygunPlayClient;
 
-public class Global extends GlobalSettings {
+public class CustomErrorHandler implements HttpErrorHandler {
 
     private String apiKey = "paste_your_api_key_here";
 
-    public Promise<Result> onError(RequestHeader request, Throwable t) {
-        RaygunPlayClient rg = new RaygunPlayClient(apiKey, request);
-        rg.SendAsync(t);
+    @Override
+    public CompletionStage<Result> onClientError(Http.RequestHeader request, int statusCode, String message) {
+        // Handle client errors if necessary. For simplicity, we'll just forward the request to Play's default error handling
+        return CompletableFuture.completedFuture(Results.status(statusCode, "A client error occurred: " + message));
```

Comment:
I am assuming you've tested all these code examples, right? And even better, added them as unit tests or something